examples/simple/root.re

```diff
@@ -1,3 +1,8 @@
 Js.log "Do you see this message? If you do, then things should work!";
 
-ReactDOMRe.render <Page message="Hello!" /> (ReasonJs.Document.getElementById "index");
+open ReasonJs;
+
+switch (document |> Document.getElementById "index") {
```

Comment:
It's nicer with an Option module. I tend to use rebase, but I don't think that's a dependency that ought to be brought in here.